numba/cuda/simulator/kernel.py

```diff
@@ -0,0 +1,180 @@
+from __future__ import print_function
+from .array import to_device
+from .kernelapi import Dim3, FakeCUDAModule, swapped_cuda_module
+from six import reraise
```

Comment:
Ah, I had forgotten we included six. The version we include (1.7.3) doesn't include `raise_from` (used elsewhere) though, so it would need to be upgraded. Is there any objection to upgrading our version to 1.9.0?